content/getting_started/custom_metrics.md

```diff
@@ -3,7 +3,7 @@ title: Custom Metrics
 kind: documentation
 ---
 
-Datadog allows for custom metrics submission through multiple ways to offer a full understanding of what is happening on your infrastructure.
+Datadog enable custom metrics submission through multiple ways for a comprehensive understanding of what is happening on your infrastructure.
```

Comment:
Overall, the old sentence and new changes are awkward. Let's update to:
"Datadog allows you to submit custom metrics in multiple ways in order to provide a comprehensive view of what is happening in your infrastructure"